rds.tf

```diff
@@ -1,9 +1,9 @@
 resource "aws_db_instance" "airflow" {
   count                     = var.postgres_uri != "" || var.airflow_executor == "Sequential" ? 0 : 1
   name                      = replace(title(local.rds_name), "-", "")
-  allocated_storage         = 20
-  storage_type              = "standard"
-  engine                    = "postgres"
+  allocated_storage         = var.rds_allocated_storage
+  storage_type              = var.rds_storage_type
+  engine                    = var.rds_engine
   engine_version            = "11.8"
```

Comment:
engine_version should also be a param

Reply:
It's part of https://github.com/datarootsio/terraform-aws-ecs-airflow/pull/24, I didn't want to steal @frederikdesmedt's thunder. Should I put everything in this one?

Reply:
nvm freddy already did it in #24